### ISSUE TITLE
Crashed at UIDevice information

### ISSUE THREAD

**Issue by JyHu** [Flipboard/FLEX#78]
Open the menu , chooice the `[UIDevice currentDevice]` row , change the segment to `Include Inheritance` , then scroll the UITableView to bottom , the app will be crashed at row fourth  IVARS(4) section .

The code at `FLEXRuntimeUtility.m` , line 210 .
```
self	Class	FLEXRuntimeUtility	0x0000000100e3dbf0
ivar	Ivar	0x198114a68	0x0000000198114a68
object	UIDevice *	0x170228740	0x0000000170228740
NSObject	NSObject		
isa	Class	UIDevice	0x000061a5952bf5b3
_numDeviceOrientationObservers	long long	3	3
_batteryLevel	float	1	1
value	id	0x0	0x0000000000000000
[0]	id		
type	const char *	"#"	0x00000001939abc00
*type	const char	'#'	'#'
```
I was tested by my iPhone 5s , mode A1533 .

**Comment by ryanolsonk** [comment]
This one is really interesting. It only happens for me on device, not in the simulator. 

Took me a while to figure it out, but I tried running with the environment variable `OBJC_DISABLE_NONPOINTER_ISA=YES` and voila, no crash. The type encoding on the `isa` ivar indicates that it's a `Class` pointer, but in iOS for arm64, that may be a lie. See http://www.sealiesoftware.com/blog/archive/2013/09/24/objc_explain_Non-pointer_isa.html for a more detailed explanation of changes to the `isa` ivar on arm64.

We probably need to call `ivar_getName()` and special case when the name is `isa`. I'll try to put together a change that resolves this.

**Comment by ryanolsonk** [comment]
Thanks for the crash report. This is now fixed in 30f1fecc54cd9ca795c2ddfaca3fae6dc6edba2f.

**Comment by JyHu** [comment]
Nice work .
